src/js/flows/ingestFiles.ts

```diff
@@ -61,15 +61,14 @@ const validateInput = (files: File[], poolNames) => ({
 
 const createPool = (client: Zealot, gDispatch, workspaceId) => ({
   async do(params: IngestParams) {
-    const pool = await client.pools.create({name: params.name})
+    const create = await client.pools.create({name: params.name})
```

Comment:
Not a blocker, but an FYI, but you could do a destructure on this object like so: `const {pool, branch} = await ...`

Reply:
Oh, that's much nicer! So much so that I did it in 7cc0fb5d11a6d551e4aea6a6d43a599b888fafdf.